config/sys.config.src

```diff
@@ -3,7 +3,7 @@
         {config_worker, #{
             host => "${HPR_CONFIG_WORKER_HOST}",
             port => "${HPR_CONFIG_WORKER_PORT:-8080}",
-            file_backup_path => "/var/data/hpr/config_worker.backup"
+            file_backup_path => "/var/data/config_worker.backup"
```

Comment:
Why?

Reply:
- remove hpr from file backup path. In docker-compose we're already mounting to a nested hpr dir, and this creates a nested dir.